sematic/api/endpoints/payloads.py

```diff
@@ -1,60 +1,66 @@
 """Augment or modify the returned json for ORM models for returns from API calls"""
 # Standard Library
 import logging
-from typing import Any, Dict, List
+from typing import Any, Dict, List, Protocol, Sequence
 
 # Sematic
-from sematic.db.models.resolution import Resolution
-from sematic.db.models.run import Run
-from sematic.db.queries import get_users
+from sematic.db.queries import get_user, get_users
 
 logger = logging.getLogger(__name__)
 
 
-def get_runs_payload(runs: List[Run]) -> List[Dict[str, Any]]:
-    """
-    Build the standard payload for a list of runs.
-    """
-    runs_payload = [run.to_json_encodable() for run in runs]
+class _JSONEncodableWithUser(Protocol):
```

Comment:
I feel this should live in `has_user_mixin.py`, as this protocol describes the `HasUserMixin` type.